src/components/browse/dataset/datasetList.tsx

```diff
@@ -1,43 +1,53 @@
 import React, { Dispatch, SetStateAction } from "react";
-import DatasetListHeader from './datasetListHeader';
-import DatasetPagination from './datasetPagination';
-import Dataset from './dataset';
-import { hitModel, searchResponseModel } from '../../../models/dataset';
-import { facetFilterModel } from '../../../models/facets'
-
+import DatasetListHeader from "./datasetListHeader";
+import DatasetPagination from "./datasetPagination";
+import Dataset from "./dataset";
+import { hitModel, searchResponseModel } from "../../../models/dataset";
+import { facetFilterModel } from "../../../models/facets";
+import PerfectScrollbar from 'react-perfect-scrollbar'
+import 'react-perfect-scrollbar/dist/css/styles.css';
 
 interface dataSetProps {
-  dsCount: number
+  dsCount: number;
   dsList: hitModel[] | null;
   setSearchResp: Dispatch<SetStateAction<searchResponseModel | null>>;
   searchKeyword: string;
   limit: number;
   setLimit: Dispatch<SetStateAction<number>>;
   filterDict: facetFilterModel[];
-};
+}
 
 const DatasetList = (props: dataSetProps) => {
-  var dsCount: number = props.dsCount
+  var dsCount: number = props.dsCount;
 
   return (
-    <div className='bg-white border p-2 px-3 rounded'>
+    <div className="bg-white border p-2 ps-3 rounded">
       <DatasetListHeader dsCount={dsCount} />
       <DatasetPagination
         setSearchResp={props.setSearchResp}
         searchKeyword={props.searchKeyword}
         limit={props.limit}
         setLimit={props.setLimit}
         dsCount={dsCount}
-        filterDict={props.filterDict} />
-      {
-        props.dsList === null ?
-          null :
-          (
-            <Dataset dsList={props.dsList} />
-          )
-      }
+        filterDict={props.filterDict}
+      />
+      {props.dsList === null ? null : (
+        <div style={{ height: "60vh"}} className="mb-2 position-relative w-100">
```

Comment:
will the hardcoded size have an impact depending on the screen size?

Reply:
I don't think we should introduce a known bug. Either we introduce the scrollbars after the research or we don't need it now. This is double the effort for no reason.

Reply:
It's not a bug, just a missing feature. Having slightly mismatched divs (a few pixels) for a bit should be ok.

Reply:
No, please take your time and provide the right solution. We can't check in hardcoded solutions.